ACKNOWLEDGEMENTS.md

```diff
@@ -21,32 +21,3 @@ distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
 implied. See the License for the specific language governing
 permissions and limitations under the License.
-
----
-
-delayed_job-rails_reloader
```

Comment:
Turns out Delayed Job automatically reloads things in development: [source](https://github.com/collectiveidea/delayed_job#development)